### PR TITLE
[PPML] Add Occlum Debug Setting

### PR DESCRIPTION
**Pull request by piaolaidelangman**
* Add Occlum debug setting
* Delete aesm.socket
* Update TpcH k8s examle
* Fix Occlum Attestation server error

### CODE REVIEW

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/executor.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ spec:
     - name: sgx-provision
       mountPath: /dev/sgx/provision
     - name: aesm
-      mountPath: /var/run/aesmd/aesm.socket
+      mountPath: /var/run/aesmd/
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/executor.yaml:17]: Remove /

File: ppml/trusted-big-data-ml/scala/docker-occlum/run_spark_on_occlum_glibc.sh [hunk 2]
```diff
@@ -44,7 +44,7 @@ init_instance() {
     new_json="$(jq '.resource_limits.user_space_size = "SGX_MEM_SIZE" |
         .resource_limits.max_num_of_threads = "SGX_THREAD" |
         .process.default_heap_size = "SGX_HEAP" |
-        .metadata.debuggable = false |
+        .metadata.debuggable = true |
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/scala/docker-occlum/run_spark_on_occlum_glibc.sh:47]: disable debug by default

File: ppml/trusted-big-data-ml/scala/docker-occlum/start-occlum-attestation-server.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@ sudo docker run -it \
 	--cpuset-cpus 5-7 \
 	--device=/dev/sgx/enclave \
 	--device=/dev/sgx/provision \
-	-v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+	-v /var/run/aesmd/:/var/run/aesmd/ \
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/scala/docker-occlum/start-occlum-attestation-server.sh:11]: We can put this aesm change into other pr

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/executor.yaml (hunk 1 above)
Comment (piaolaidelangman) [reply ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/executor.yaml]: Cancel aesm change.

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**